### PR TITLE
Fix/#redirect error

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
Issue Title: Redirect Error When Clicking "Tools" #142

Fix Redirect Error.

f2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,13 +51,19 @@ You are responsible for:
...
-### Features
 
-![javascript][]
-![react][]
+|                                                   |
+| :------------------------------------------------------------------------------------------------------: |
+|                                         Posts API:                       |
+|                          <https://jsonplaceholder.typicode.com/posts>                    |
+|                                         Comments API:                       |
+|                          <https://jsonplaceholder.typicode.com/comments>                   |
```
Comment (Israel-Laguan) [inline README.md]: I think a list would be shorter instead of a table

**Review by e-azocar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Israel-Laguan) [reply README.md]:
```suggestion
|
| :---:
| Posts API:
| <https://jsonplaceholder.typicode.com/posts>
| Comments API:
|<https://jsonplaceholder.typicode.com/comments>
```

Comment (mdrianislam0or1) [reply README.md]: f1
f2

should i push it? is it ok?

Comment (Israel-Laguan) [reply README.md]: I meant to erase some extra spaces in this one

Comment (mdrianislam0or1) [reply README.md]: erase extra space.
f5
s

**Review by Israel-Laguan — APPROVED** [review]

**Review by angelik0828 — APPROVED** [review]

**Pull request merged**